homeassistant/components/slide/cover.py

```diff
@@ -0,0 +1,128 @@
+"""Support for Go Slide slides."""
+
+import logging
+
+from homeassistant.util import slugify
+from homeassistant.components.cover import (
+    ATTR_POSITION,
+    ENTITY_ID_FORMAT,
+    SUPPORT_CLOSE,
+    SUPPORT_OPEN,
+    SUPPORT_SET_POSITION,
+    SUPPORT_STOP,
+    STATE_OPEN,
+    STATE_CLOSED,
+    DEVICE_CLASS_CURTAIN,
+    CoverDevice,
+)
+from .const import API, DOMAIN, SLIDES
+
+_LOGGER = logging.getLogger(__name__)
+
+
+# pylint: disable=unused-argument
+async def async_setup_platform(hass, config, async_add_entities, discovery_info=None):
+    """Set up cover(s) for Go Slide platform."""
+    entities = []
```

Comment:
Add a guard clause that checks if discovery_info is None and return if so.

Reply:
Made the change.